Podfile.lock

```diff
@@ -52,6 +52,6 @@ SPEC CHECKSUMS:
   SwiftLint: 3207c1faa2240bf8973b191820a116113cd11073
   Turf: 9763a5a2d7da05dfdd5ed56aeb0414b291b0fee1
 
-PODFILE CHECKSUM: 7710c091be957106e25f3d26bb8ae6b31dae2ee7
+PODFILE CHECKSUM: a7235f149d0c9b1035f598f2965925c3ad9321c7
 
-COCOAPODS: 1.6.0.beta.1
+COCOAPODS: 1.6.0.beta.2
```

Comment:
What does this mean?

Reply:
When one does `bundle install`, it’ll install CocoaPods 1.6.0.beta.2 now, instead of beta.1.

Reply:
Errr, I misread the file name — since this is `Podfile.lock`, this line refers to the version of CocoaPods that was used to install these pods.